openreview/conference/templates/bidWebfield.js

```diff
@@ -345,5 +321,58 @@ function addEdgesToNotes(validNotes, edgesMap) {
   return validNotes;
 }
 
+function renderNotes(notes) {
+
+
+    // Render the contents of the All Papers tab
+    var searchResultsOptions = _.assign({}, paperDisplayOptions, { container: '#allPapers' });
+    var submissionListOptions = {
+      heading: null,
+      container: '#allPapers',
+      search: {
+        enabled: true,
+        localSearch: false,
+        subjectAreas: SUBJECT_AREAS,
+        subjectAreaDropdown: 'basic',
+        invitation: BLIND_SUBMISSION_ID,
+        sort: false,
+        onResults: function(searchResults) {
+          Webfield.ui.searchResults(prepareNotes(searchResults, conflictIds, bidsByNote), searchResultsOptions);
+        },
+        onReset: function() {
+          Webfield.ui.searchResults(notes, searchResultsOptions);
+          $('#allPapers').append(view.paginationLinks(noteCount, 50, 1));
+        },
+      },
+      displayOptions: paperDisplayOptions,
+      autoLoad: false,
+      noteCount: noteCount,
+      pageSize: 50,
+      onPageClick: function(offset) {
+        return getPapersSortedByAffinity(offset, selectedScore)
+        .then(function(notes) {
+          return prepareNotes(notes, conflictIds, bidsByNote);
+        });
+      },
+      fadeIn: false
+    };
+
+    Webfield.ui.submissionList(notes, submissionListOptions);
+    var affinitySelectedClass = selectedScore == 'Affinity' && 'selected';
+    var tpmsSelectedClass = selectedScore == 'TPMS' && 'selected';
+
+    $('#notes .form-inline.notes-search-form').append('<div class="form-group score"><label for="score-dropdown">Score:</label></div><select class="score-dropdown form-control"><option value="Affinity" ' + affinitySelectedClass + '>Affinity</option><option value="TPMS" ' + tpmsSelectedClass + '>TPMS</option></select>');
+}
+
 // Go!
 main();
+
+// Event handlers
+$('#invitation-container').on('change', 'form.notes-search-form .score-dropdown', function(e) {
+  console.log('change!', $(this).val());
+  selectedScore = $(this).val();
+        return getPapersSortedByAffinity(0, selectedScore)
+        .then(function(notes) {
+          return renderNotes(prepareNotes(notes, conflictIds, bidsByNote));
```

Comment:
replacing renderNotes with:

```
          $('.submissions-list', '#allPapers').replaceWith(
            Handlebars.templates['partials/noteList']({
              notes: prepareNotes(notes, conflictIds, bidsByNote),
              options: paperDisplayOptions
            })
          );
```

seems to fix the issues with the event but it doesn't reload the pagination bar, so it keeps in the same page. I'm not sure if we should reset the offset value to 0.